packages/tools/kadena-cli/src/utils/test.util.ts

```diff
@@ -18,11 +18,14 @@ class OutputCapture {
   }
   public capture(print: boolean = false): void {
     this._print = print;
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
     process.stdout.write = (chunk, encoding?: any, callback?: any) => {
       this._stdout.push(chunk.toString());
       if (this._print) this._originalStdoutWrite(chunk, encoding, callback);
       return true;
     };
+
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
     process.stderr.write = (chunk, encoding?: any, callback?: any) => {
```

Comment:
We can use `unknown` since it covers the intent. We don't know what it is that we get passed here.